### PR TITLE
DrawBuffers has no effect on sampling feedback.

### PR DESCRIPTION
**Pull request by jdashg**
Incomplete textures should be ignored by feedback detection also.

### CODE REVIEW

**Comment by jdashg** [comment]
This should resolve #2670.

**Comment by jdashg** [comment]
Actually since cubemap attachments must be cube-complete, we should be able to always do this detection properly, even on mipmap-incomplete textures.
That said, it would be easy enough to excuse incomplete textures as impossible "to sample from the texture object T bound to texture unit U", and therefore skipped for the purposes of feedback.

File: sdk/tests/conformance/rendering/rendering-sampling-feedback-loop.html [hunk 1]
```diff
@@ -62,82 +83,102 @@
...
+    }
+
     init();
 
     // The sampling texture is bound to COLOR_ATTACHMENT1 during resource allocation
     allocate_resource();
 
-    rendering_sampling_feedback_loop([gl.NONE, gl.COLOR_ATTACHMENT1], gl.INVALID_OPERATION);
-    rendering_sampling_feedback_loop([gl.COLOR_ATTACHMENT0, gl.COLOR_ATTACHMENT1], gl.INVALID_OPERATION);
-    rendering_sampling_feedback_loop([gl.COLOR_ATTACHMENT0, gl.NONE], gl.NO_ERROR);
```
Comment (kainino0x) [inline sdk/tests/conformance/rendering/rendering-sampling-feedback-loop.html]: Is it possible to keep a [gl.COLOR_ATTACHMENT0] test case too? I know it's not supposed to make a difference, but just for coverage.

Or maybe keep all of the original cases (with two attachments), if there's no particular need to simplify.

File: sdk/tests/conformance/rendering/rendering-sampling-feedback-loop.html [hunk 2]
```diff
@@ -62,82 +83,102 @@
...
+        rendering_sampling_feedback_loop([gl.NONE]);
+    }
 }
 
 function init() {
-    program = wtu.setupProgram(gl, ['vshader', 'fshader'], ['aPosition', 'aTexCoord'], [0, 1]);
-    positionLoc = gl.getAttribLocation(program, "aPosition");
-    texCoordLoc = gl.getAttribLocation(program, "aTexCoord");
+    let shaders = ["vs100", "fs100"];
+    if (gl.texStorage2D) {
```
Comment (kainino0x) [inline sdk/tests/conformance/rendering/rendering-sampling-feedback-loop.html]: Is this a version check? Don't we have a wtu thing for "is webgl2"? `getDefault3DContextVersion() === 2` I guess

Comment (jdashg) [reply sdk/tests/conformance/rendering/rendering-sampling-feedback-loop.html]: Yeah, but it's easier to do it this way, and it always works directly on whatever `gl` is passed in. TBH a gl.version() would be nice to inject when we create a context, but this works.

File: sdk/tests/conformance/rendering/rendering-sampling-feedback-loop.html (hunk 1 above)
Comment (jdashg) [reply sdk/tests/conformance/rendering/rendering-sampling-feedback-loop.html]: Sure.

**Comment by kenrussell** [comment]
LGTM after the fact. Thanks @kainino0x for reviewing this.

**Pull request merged**